applications/rag/workloads.tfvars

```diff
@@ -14,8 +14,13 @@
 
 project_id = "<your project ID>"
 
+# GCP Network configuration
+create_network  = false
```

Comment:
not sure we want to expose these.. then we also need to expose the CIDR range fields which makes things complex. How about just using default VPC for RAG and requiring the user to enable PSC?

wdyt @umeshkumhar

Reply:
>  then we also need to expose the CIDR range fields which makes things complex

What if we just update `infrastructure` to auto create subnets? Then we can defer to GCP defaults for subnets, and we don't need to worry about.

This also fixes a problem where the the GKE cluster is created in a different region from the CloudSQL instance. Today this wouldn't work with a `create_network = true`, since custom-network only creates a single subnet region.

Reply:
I think that should work - does TF support auto creating subnets?

Reply:
TF does, but the auto created subnets do not have "Private Google Access" enabled. I've updated the script to create CIDR sub-ranges dynamically from a provided CIDR range. This effectively gives us auto subnets, but with the ability to tune their parameters.

Reply:
yes its still the same, we are now creating multiple subnets in multiple regions.

but we still need inputs cidr range, newbits, list of locations.. 

Rather to keep it simple, I would suggest to be it in single region for now. and allow inputs for cidr range and region.. and set them default..
https://github.com/GoogleCloudPlatform/ai-on-gke/blob/b8b72a9ded1d2e0b3cf587a9b393b2a9eca2296f/infrastructure/platform.tfvars#L24

we assume and also recommended that SQL instance and GKE lies in same region.